Automattic/jetpack
# Simple Payments: Preserve white-space in Pay with Paypal product description

Fixes #17035

#### Changes proposed in this Pull Request:

* Preserves white-space entered in the description field of the Simple Payments block, instead of collapsing it.

Before:
<img width="554" alt="Screen Shot 2021-02-05 at 4 14 06 PM" src="https://user-images.githubusercontent.com/63313398/107102050-177e2900-67ce-11eb-91f4-2c998c337d40.png">

After:
<img width="555" alt="Screen Shot 2021-02-05 at 4 14 54 PM" src="https://user-images.githubusercontent.com/63313398/107102043-0fbe8480-67ce-11eb-843a-7ec84f42a8c5.png">


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Create a new post and add a Pay with Paypal block
* In the description field, write a message containing line breaks
* View the post on the frontend and verify that the line breaks have been preserved


**NOTE:** When testing on wpcom, you will need to make sure that the changes to `simple-payments.css` sync to the correct location in the sandbox. That is `wp-content/mu-plugins/simple-payments/simple-payments.css`.

#### Proposed changelog entry for your changes:


*
